charts/mod-data-export/values.yaml

```diff
@@ -80,6 +80,44 @@ jmx:
 
 heapDumpEnabled: false
 
+volumeClaims:
+  - enabled: false
```

Comment:
false???

Reply:
Because we need to allocate additional volume specifically for the perf-firebird and testing-sprint environments, this additional volume will be disabled by default and will be enabled for those environments via the pipeline-shared-library.